loops_lab.js

```diff
@@ -0,0 +1,154 @@
+//Loops lab Assignment
```

Comment:
Overall good demonstration of how to use for and while loops. I like the fact that you tried to solve problems using different approaches. Keep on practicing!